ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -387,6 +420,99 @@ FileSpan addDeleteEntry(MockId idToDelete, MessageInfo info) throws StoreExcepti
     return fileSpan;
   }
 
+
+
+  /**
+   * Adds a delete entry in the index (real and reference) for {@code idToDelete}.
+   * @param idToDelete the id to be deleted.
+   * @return the {@link FileSpan} of the added entries.
+   * @throws IOException
+   * @throws StoreException
+   */
+  FileSpan addUndeleteEntry(MockId idToDelete) throws IOException, StoreException {
```

Comment:
They are not used. Is this a placeholder?

Reply:
I added this anticipating that I'd need to use it for building out tests, but it ended up I didn't.  Still seems like something that should be in there though for compaction tests later